ui/packages/catalog-ui-search/src/main/webapp/react-component/location/keyword.js

```diff
@@ -29,6 +29,12 @@ class Keyword extends React.Component {
       loading: false,
       error: null,
       polyType: null,
+      minimumInputLength: props.minimumInputLength || 2,
+      placeholder: props.placeholder || 'Pan to a region, country, or city',
+      renderLoadingDisplay:
```

Comment:
:pencil2: I don't think these variables need to be added to state since they can be derived from props

Reply:
Agreed. I added a new commit that moves them out.